doc/RecordingControl.xml

```diff
@@ -1873,12 +1873,14 @@ SetRecordingJobMode(JobToken, Active)
     </section>
     <section xml:id="_refCloudRecordingSegments">
       <title>Segments</title>
-      <para>
-        When a span is opened, the device shall also open a new segment for this span.
-        Inversely, when a span is closed, the device shall close the open segment of this span.
-        When a span is open and the duration of the open segment of this span reaches the <literal>SegmentDuration</literal> configured for the recording, the device shall close the open segment of this span and open a new segment for this span.
-        When closing the open segment of a span and opening a new segment for this span, there shall be no time gap between the two segments.
-      </para>
+      <para> When a span is opened, the device shall also open a new segment for this span.
+        Inversely, when a span is closed, the device shall close the open segment of this span. When
+        a span is open and the duration of the open segment of this span reaches the
+          <literal>SegmentDuration</literal> configured for the recording, the device shall close
+        the open segment of this span and open a new segment for this span. When closing the open
+        segment of a span and opening a new segment for this span, there shall be no time gap
+        between the two segments. Note that the actual segment duration may exceed the configured
+        duration because a segment must start with an I-Frame.</para>
```

Comment:
Can the actual segment duration exceed, equal, or be less than the configured duration, or is it only allowed to exceed or equal the configured duration?

Reply:
Based on the 12/5 telco, please check the suggested text below.

"Note that the actual segment duration may vary and can be plus or minus one GOP size from the configured duration because a segment must start with an I-Frame."